package.json

```diff
@@ -1,93 +1,103 @@
 {
-  "name": "@balena/open-balena-api",
-  "description": "Internet of things, Made Simple",
-  "version": "0.18.0",
-  "license": "AGPL-3.0",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/balena-io/open-balena-api"
-  },
-  "main": "dist/index",
-  "files": [
-    "dist/"
-  ],
-  "scripts": {
-    "build": "npm run clean && tsc --project ./tsconfig.build.json && copyup \"src/**/*.sbvr\" \"src/**/*.sql\" dist/",
-    "clean": "rimraf dist/",
-    "lint": "resin-lint --typescript src/ init.ts && tsc --noEmit --project .",
-    "prettify": "prettier --config ./node_modules/resin-lint/config/.prettierrc --write index.js init.ts \"src/**/*.ts\" \"typings/**/*.ts\"",
-    "prepare": "npm run build",
-    "test": "npm run build && npm run lint"
-  },
-  "dependencies": {
-    "@resin.io/device-types": "^10.5.0",
-    "@resin/pinejs": "^10.11.0",
-    "@types/basic-auth": "^1.1.2",
-    "@types/bluebird": "^3.5.25",
-    "@types/cookie-session": "^2.0.36",
-    "@types/express": "^4.16.1",
-    "@types/express-brute": "0.0.35",
-    "@types/express-brute-redis": "0.0.0",
-    "@types/json-schema": "^7.0.3",
-    "@types/jsonwebtoken": "^8.3.2",
-    "@types/lodash": "^4.14.122",
-    "@types/memoizee": "^0.4.2",
-    "@types/node": "^8.10.45",
-    "@types/node-uuid": "0.0.28",
-    "@types/passport-jwt": "^3.0.1",
-    "@types/randomstring": "^1.1.6",
-    "@types/raven": "^2.5.3",
-    "@types/redis": "^2.8.12",
-    "@types/request": "^2.48.1",
-    "array-sort": "^1.0.0",
-    "avsc": "^5.4.7",
-    "aws-sdk": "^2.438.0",
-    "basic-auth": "^2.0.1",
-    "bluebird": "^3.5.4",
-    "body-parser": "^1.18.3",
-    "compression": "^1.7.4",
-    "cookie-parser": "^1.4.4",
-    "cookie-session": "^1.3.3",
-    "express": "^4.16.4",
-    "express-brute": "^1.0.1",
-    "express-brute-redis": "0.0.1",
-    "fast-boot2": "^1.0.9",
-    "ipaddr.js": "^1.9.0",
-    "json-schema": "^0.2.3",
-    "jsonwebtoken": "^8.5.1",
-    "lodash": "^4.17.10",
-    "memoizee": "^0.4.14",
-    "method-override": "^3.0.0",
-    "ndjson": "1.5.0",
-    "node-uuid": "~1.4.0",
-    "passport": "^0.4.0",
-    "passport-jwt": "^4.0.0",
-    "randomstring": "^1.1.5",
-    "raven": "^2.6.4",
-    "redis": "^2.8.0",
-    "request": "^2.85.0",
-    "resin-device-config": "^5.1.1",
-    "resin-semver": "^1.5.1",
-    "supervisor": "^0.12.0",
-    "thirty-two": "^1.0.0",
-    "ts-node": "^7.0.1",
-    "typed-error": "^2.0.0",
-    "typescript": "^3.4.3"
-  },
-  "devDependencies": {
-    "copyfiles": "^2.1.0",
-    "husky": "^1.3.1",
-    "lint-staged": "^8.1.5",
-    "prettier": "^1.17.0",
-    "resin-lint": "^2.0.0",
-    "rimraf": "^2.6.3"
-  },
-  "engines": {
-    "node": ">=8.0"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  }
+	"name": "@balena/open-balena-api",
```

Comment:
Could you have npm auto-format this please

Reply:
yeah - not sure what happened here, but I was planning to sort it out